### ISSUE TITLE
#341278580------------- Elements must have sufficient color contrast

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#670]
<strong>Source:</strong>`<span id="18a489aab27-widget-9_18a489aab16-object-96__title" title="People (Invisible at current scale)" aria-label="People (Invisible at current scale)" class="esri-layer-list__item-title">People</span>`
<strong>Element Location: </strong>#\31 8a489aab27-widget-9_18a489aab16-object-96__title
<strong>Impact: </strong>serious
<strong>failureSummary:</strong> Fix any of the following:
  Element has insufficient color contrast of 1.72 (foreground color: #c5c5c5, background color: #ffffff, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1
Tags: cat.color,wcag2aa,wcag143,ACT
Screenshot:`C:\Users\khy12817\Desktop\Test.PNG`
